starboundconfig.json

```diff
@@ -0,0 +1,144 @@
+[
+  {
+    "DisplayName": "Server Name",
+    "Category": "Starbound - Settings",
+    "Description": "The name of the server.",
+    "Keywords": "Server, name",
+    "FieldName": "Name",
+    "InputType": "text",
+    "IsFlagArgument": false,
+    "ParamFieldName": "serverName",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "AMP Starbound Server",
+    "EnumValues": {}
+  },
+  {
+    "DisplayName": "Max Players",
+    "Category": "Starbound - Settings",
+    "Description": "Max number of players connected at one time.",
+    "Keywords": "max,players,users",
+    "FieldName": "$MaxUsers",
+    "InputType": "number",
+    "IsFlagArgument": false,
+    "ParamFieldName": "maxPlayers",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "Placeholder":  "8"
+  },
+  {
+    "DisplayName": "Server Port",
+    "Category": "Starbound - Settings",
+    "Description": "UDP port for game traffic.",
+    "Keywords": "",
+    "FieldName": "$gameServerPort",
+    "InputType": "number",
+    "IsFlagArgument": false,
+    "ParamFieldName": "gameServerPort",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "EnumValues": {},
+    "Hidden": true
+  },
+  {
+    "DisplayName": "Query Port",
+    "Category": "Starbound - Settings",
+    "Description": "UDP port for Steam server list features.",
+    "Keywords": "",
+    "FieldName": "$gameServerPort",
+    "InputType": "number",
+    "IsFlagArgument": false,
+    "ParamFieldName": "queryServerPort",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "EnumValues": {},
+    "Hidden": true
+  },
+  {
+    "DisplayName": "RCON Port",
+    "Category": "Starbound - Settings",
+    "Description": "UDP port for RCON connection.",
+    "Keywords": "",
+    "FieldName": "$rconServerPort",
+    "InputType": "number",
+    "IsFlagArgument": false,
+    "ParamFieldName": "rconServerPort",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "EnumValues": {},
+    "Hidden": true
+  },
+  {
+    "DisplayName": "RCON Password",
+    "Category": "Starbound - Settings",
+    "Description": "RCON Password.",
+    "Keywords": "",
+    "FieldName": "$RemoteAdminPassword",
+    "InputType": "password",
+    "IsFlagArgument": false,
+    "ParamFieldName": "rconServerPassword",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "EnumValues": {},
+    "Hidden": true
+  },
+  {
+    "DisplayName": "RCON Enabled",
+    "Category": "Starbound - Settings",
+    "Description": "RCON Password.",
```

Comment:
Doesn't strictly matter since it is hidden, but the description is incorrect

Reply:
Addressed as well as changed the category for easy spotting in the config manifest.